xla/service/gpu/backend_configs.proto

```diff
@@ -56,6 +56,8 @@ message CudnnConvBackendConfig {
     // layout stays the same, but the data is shuffled in a way that is
     // compatible with NVidia's IMMA instruction (sm75+).
     bool reordered_int8_nchw_vect = 7;
+
+    string call_context = 10;
```

Comment:
Is it intentional to put the call_context inside the filter_and_bias_reordering_oneof?

Reply:
Moved it outside